modules/enet/networked_multiplayer_enet.cpp

```diff
@@ -401,28 +405,30 @@ void NetworkedMultiplayerENet::poll() {
 							}
 
 						} else if (target < 0) {
-							// To all but one
+							
+							// Send to all but one if server_relay is enabled	
+							if (server_relay) {
+								// And make copies for sending
+								for (Map<int, ENetPeer *>::Element *E = peer_map.front(); E; E = E->next()) {
 
-							// And make copies for sending
-							for (Map<int, ENetPeer *>::Element *E = peer_map.front(); E; E = E->next()) {
-
-								if (uint32_t(E->key()) == source || E->key() == -target) // Do not resend to self, also do not send to excluded
-									continue;
+									if (uint32_t(E->key()) == source || E->key() == -target) // Do not resend to self, also do not send to excluded
+										continue;
 
-								ENetPacket *packet2 = enet_packet_create(packet.packet->data, packet.packet->dataLength, packet.packet->flags);
+									ENetPacket *packet2 = enet_packet_create(packet.packet->data, packet.packet->dataLength, packet.packet->flags);
 
-								enet_peer_send(E->get(), event.channelID, packet2);
+									enet_peer_send(E->get(), event.channelID, packet2);
+								}
 							}
-
+							
```

Comment:
There is trailing whitespace on this line.

Reply:
Fixed